src/lib/comms/include/sol-network.h

```diff
@@ -73,6 +73,9 @@ extern "C" {
  */
 #define SOL_INET_ADDR_STRLEN 48
 
+#ifndef AF_BT_IOTIVITY
+    #define AF_BT_IOTIVITY 31
```

Comment:
This socket is made to be able to comunicate with Iotivity's instances and Soletta's instances. It contains the very same characteristics and services of Iotivity.

@glima, @vcgomes and I were talking about this, and since this patch is not for a general puporse, it would be nice to expose it as the name iotivity. 

I was thinking now I should also have pointed this out in the commit message.